docs/concepts/ksql-and-kafka-streams.rst

```diff
@@ -0,0 +1,111 @@
+.. _ksql-and-kafka-streams:
+
+KSQL and Kafka Streams
+######################
+
+KSQL is the streaming SQL engine for Apache Kafka®. With KSQL, you can write
+real-time applications by using a SQL-like query language.
+
+Kafka Streams is the Apache Kafka® library for writing real-time applications
+and microservices in Java and Scala.
+
+KSQL is built on Kafka Streams and occupies the top of the stack in |cp|.
+
+.. image:: ../img/ksql-kafka-streams-core-kafka-stack.png
+
+KSQL gives you the highest level of abstraction for implementing real-time
+business logic on Kafka topics. KSQL automates much of the complex programming
+that's required for real-time operations on streams of data, so that one line
+of KSQL can do the work of a dozen lines of Java and Kafka Streams.
+
+For example, to implement fraud-detection logic on a Kafka topic named

```

Comment:
Maybe reword it so it shows that we know this is a very simple `fraud detection` approach!

Reply:
Also you may want to include the function for fraud probability in the query. Maybe something like this:

`CREATE STREAM fraudulent_payments AS SELECT fraudProbability( data) FROM payments
     WHERE fraudProbability( data) > 0.8;`